charts/dapr/charts/dapr_placement/templates/dapr_placement_statefulset.yaml

```diff
@@ -177,9 +175,16 @@ spec:
                 fieldPath: metadata.namespace
       serviceAccountName: dapr-placement
       volumes:
-        - name: credentials
-          secret:
-            secretName: dapr-trust-bundle
+      - name: dapr-trust-bundle
+        configMap:
+          name: dapr-trust-bundle
+      - name: dapr-identity-token
```

Comment:
What about kubelets that don't supported projected volumes? Is there a way to make this optional (so users can configure Sentry to use JWTs signed somewhere else instead)?

Reply:
We have at least one large customer who depends on a kubelet without support for projected volumes.

Reply:
OK, can we do this in another PR as this is going to effect all the other components anyway so will be cleaner to do all of them at once in a dedicated PR?

Reply:
Ok, can you please open an issue so we don't forget this is a P0?

Reply:
I've added it to the list of the parent issue https://github.com/dapr/dapr/issues/5756